### ISSUE TITLE
Keras CNN slower predicting than equivalent tensorflow code

### ISSUE THREAD

**Issue by lminer** [keras-team/keras#8379]
I'm using a CNN for text classification with around 230 classes. I've tried switching the code over to Keras, but have found that the code is three times slower in predicting than the equivalent tensorflow. It was only 1.5 times slower when I used tensorflow for the final softmax layer and saved the model as a tensorflow model rather than a keras model.

Any ideas on further bits I should try moving to tensorflow? The architecture is word embeddings > dropout > 1D convolutional layer with  1D max pooling > dropout > 2 Dense > dropout > TF softmax.

I saw that embedding layer could be slow from [this issue](https://github.com/fchollet/keras/issues/4365), but this seems to be a training time slowdown.

**Comment by Derekkk** [comment]
Yea, I got the same problem. Don't have the specific running time data though, but Keras predicting time seems much much slower than TF when I test QPS using the same network structure. Anyone knows about it? Massive thanks.
